README.md

```diff
@@ -1 +1,46 @@
-# Hello-world
\ No newline at end of file
+Project name : Hello Microverse
+Tech Stack:
+    - this project use html and css and javascript
+    - apply linters to support code optimization
+
+Key feature:
+    - .gitignore
+    - linters
+
+GETTING STARTED
+Prequesites: none
+Setup: clone this repo to your desktop
+Intall: none
+Usage: free style
+Run tests: none
+Deployment: none
+
+AUTHORS:
+    1. Khanh (Mr) 
+        - Github: khanh-devos, 
+        - Twitter: khanhror
+        - LinkedIn: Khanh Than Trong
+
+Future Features:
+    - add some specific content to it.
+    - implement responsive.
+    - use javascript to add some dynamic feature. 
+
+Contribution:
+    - All contributions are welcome
+
+Supports:
+    - If you like this project, can invite me a cup of tea
+
+Acknowlegments:
+    I would like to thank to everybody at Microverse
+
+License:
+    This project is MIT licensed.
+
```

Comment:
- [ ] As requested by the previous reviewer, it's recommended to use the [README template](https://github.com/microverseinc/readme-template) for your repository's `README`. You can easily copy the template by following these steps:

### Step 1: Go to [this link](https://github.com/microverseinc/readme-template) and click on the README file as in the image:
![image](https://user-images.githubusercontent.com/107952724/234482413-a83503c4-4afe-4506-a588-64722805daed.png)

### Step 2: Click on the copy raw file or if you want you can download it and paste it into your project.
![image](https://user-images.githubusercontent.com/107952724/234482958-89a99a38-71c4-498b-87cc-870192cbf229.png)

### Step 3: Customize the template with your project information, and also don't forget to remove the commented code and the logo image. 👌

Following these steps will help you create a more professional `README` file.

Reply:
Hello Diego Garcia, 
thanks alot for your specific review